book/plugins/android/single-bundle.md

```diff
@@ -0,0 +1,34 @@
+## Single Bundle React Naitve
+
+![single-bundle.png](./single-bundle.png)
+
+### OVERVIEW
+This plugin gives ability to change version of react native and react. It also aggregate all react native plugins into one single bundle and upload it to S3. Plugins becomes modules under the names specified as plugin identifiers from plugin configuration."
+### PLUGINS PREPERATIONS
+
+#### Add Single Bundle Plugin to project configuration
+Add single bundle plugin and select versions of react-native, react and react-native-zapp-bridge you want to use. Save plugin configuration and rebuild. Please note rebuild is necesery every time you change versions or adding/removing other react native plugins.
+
+![Single Bundle Configuration]( https://assets-production.applicaster.com/applicaster-employees/zapp_team/anna_bauza/react_native/single-bundle-config.png  "Single Bundle Configuration")
+
+#### Make sure plugins support selected versions
+Your React Native plugins need to be build and published to NPM for selected versions of react and react-native.
+
+
+#### Migrate your plugins
+In order to make Single Bundle aggregator work please make sure all your react native plugins has been migrated and published to npm as following [document](https://docs.google.com/document/d/16d3b3eah5frSaz1zatefVs9FJQFAEJT8Ip4utY0Z__s/edit) instruct.
```

Comment:
Please change the link to a non edit link (as some external developers would use it).
Preferably some share link with read only permissions

Reply:
I did